islandora_transforms/BPL_MODS_to_solr.xslt

```diff
@@ -139,6 +154,28 @@
     <xsl:apply-templates mode="slurp_for_bpl_subject_descendants" select="."/>
   </xsl:template>
 
+  <!-- Concatenate filing-suffix and call-number with space. -->
+  <xsl:template mode="slurp_for_bpl" match="mods:mods">
+    <xsl:variable name="content">
+      <xsl:choose>
```

Comment:
Verified with Stephen, the suffix should be before the call number, and I realized it's silly to put the suffix in the call number field when no call number is present, as there is already a field for the suffix in the metadata display.